BredaUniversityGames/Y2024-25-PR-BB
# Fix/bloom-lightning-pass

### Description

Brightness is now correctly loaded during the sky dome pass. Previously the brightness sampled from the lighting pass was discarded and overridden by the sky dome pass.

![image](https://github.com/user-attachments/assets/72dbfa3f-9c10-412d-8bf2-9fde11e0bcc8)

### Issues

_Provide a list of related issues that should be closed_

### Test criteria

1. Open the program
2. Look at the chessboard
3. Play around with the bloom strength
4. Notice that models also become brighter whenever the bloom strength increases
